Telemetry encryption in database (#1485)
guardicore/monkey: CHANGELOG.md

- Overlapping Guardicore logo in the landing page. #1441
- PBA table collapse in security report on data change. #1423
- Unsigned Windows agent binaries in Linux packages are now signed. #1444

> - Some of the gathered credentials no longer appear in database plaintext. #1454

Reviewer: ```suggestion
- Credentials gathered from victim machines are no longer stored in plaintext in the database. #1454
```

^ This should move to the **Security** section of the changelog
